Makefile

```diff
@@ -116,6 +116,7 @@ tfp: vendor assets-amp
 	$(EXEC) $(CONSOLE) doctrine:database:create --if-not-exists --env=test_mysql || true
 	$(EXEC) $(CONSOLE) doctrine:schema:drop --force --env=test_mysql || true
 	$(EXEC) $(CONSOLE) doctrine:schema:create --env=test_mysql || true
+	$(EXEC) $(CONSOLE) doctrine:query:sql "ALTER TABLE groups CHANGE address_country address_country VARCHAR(2) DEFAULT NULL" --env=test_mysql || true
```

Comment:
Why ? Can you create a doctrine migration instead ?

Reply:
tests via makefile does not run migrations:
`doctrine:schema:create --env=test_mysql`

Reply:
So the schema is not sync with the model.

Reply:
indeed. another PR will fix this